From: Greg Banks <gnb@sgi.com>
Subject: export node_to_cpu_mask to allow SGI access to it.
Patch-mainline: never
References: 182109

Why don't people ever submit their external modules to mainline so we
don't have to worry about little things like this...

Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 arch/ia64/kernel/numa.c |    1 +
 1 files changed, 1 insertion(+)

--- linux-2.6.16.orig/arch/ia64/kernel/numa.c
+++ linux-2.6.16/arch/ia64/kernel/numa.c
@@ -29,6 +29,7 @@ u8 cpu_to_node_map[NR_CPUS] __cacheline_
 EXPORT_SYMBOL(cpu_to_node_map);
 
 cpumask_t node_to_cpu_mask[MAX_NUMNODES] __cacheline_aligned;
+EXPORT_SYMBOL_GPL(node_to_cpu_mask);
 
 /**
  * build_cpu_to_node_map - setup cpu to node and node to cpumask arrays
